### PR TITLE
maint: 2022.2 dev bump

### PR DESCRIPTION
**Pull request by lizgehret**

### CODE REVIEW

File: 2022.2/staged/core/conda_build_config.yaml [hunk 1]
```diff
@@ -4,6 +4,60 @@ pandas:
...
+q2_composition:
+- 2022.2.0.dev0
+q2_cutadapt:
+- 2022.2.0.dev0
+q2_dada2:
+- 2022.2.0.dev0+1.gae2607f
+q2_deblur:
+- 2022.2.0.dev0
+q2_demux:
+- 2022.2.0.dev0
```
Comment (thermokarst) [inline 2022.2/staged/core/conda_build_config.yaml:18]:
```suggestion
- 022.2.0.dev0+1.g528bc07
```

File: 2022.2/staged/core/conda_build_config.yaml [hunk 2]
```diff
@@ -4,6 +4,60 @@ pandas:
...
+q2_phylogeny:
+- 2022.2.0.dev0
+q2_quality_control:
+- 2022.2.0.dev0
+q2_quality_filter:
+- 2022.2.0.dev0
+q2_sample_classifier:
+- 2022.2.0.dev0
+q2_taxa:
+- 2022.2.0.dev0
```
Comment (thermokarst) [inline 2022.2/staged/core/conda_build_config.yaml:48]:
```suggestion
- 2022.2.0.dev0+1.g8e9ad83
```

**Review by thermokarst — COMMENTED** [review]
Please do not merge/accept these suggestions until all of these package versions are up on Library, in the tested channel.

File: 2022.2/staged/core/conda_build_config.yaml [hunk 3]
```diff
@@ -4,6 +4,60 @@ pandas:
...
+q2_composition:
+- 2022.2.0.dev0
+q2_cutadapt:
+- 2022.2.0.dev0
+q2_dada2:
+- 2022.2.0.dev0+1.gae2607f
+q2_deblur:
+- 2022.2.0.dev0
+q2_demux:
+- 022.2.0.dev0+1.g528bc07
```
Comment (lizgehret) [inline 2022.2/staged/core/conda_build_config.yaml:18]:
```suggestion
- 022.2.0.dev0+1.g528bc07
+ 2022.2.0.dev0+1.g528bc07
```

**Pull request merged**